pages/common/carp.md

```diff
@@ -0,0 +1,28 @@
+# carp
+
+> REPL and build tool for Carp.
+> More information: <https://carp-lang.github.io/carp-docs/Manual.html>.
+
+- Start a REPL:
+
+`carp`
+
+- Start a REPL with a custom prompt:
+
+`carp --prompt "> "`
+
+- Build a file:
```

Comment:
```suggestion
- Build a `.carp` file:
```
I think it can't hurt to be a bit more specific here. Same for the rest of the examples.